web-platform/website/src/components/pages/studies/study-card/ToastComponent.tsx

```diff
@@ -16,9 +16,10 @@ interface ToastProps {
   button: string;
   type: string;
   close?: string;
-  openModal?: () => void;
+  openEmailModal?: () => void;
```

Comment:
Why context specific name? Since this component is generic we should use generic names. We should also name this onOpenModal to better indicate it being a callback.